### DISCUSSION TITLE
Feature Request: Manual review interval

### DISCUSSION THREAD

**Discussion by TaylorHuston** [st3v3nmw/obsidian-spaced-repetition#919] [General]
I'd like to be able to specify a manual interval to review some items. Specifically, for my use case, I'd like to be able to designate some notes as say "review every 30 days" or something. as opposed to the review frequency being determined by an algorithm.

**Comment by GallashZhukau** [comment]
Me too. We have declarative memory and procedural memory. The first one is about formulas, dates, and everything we can say or write, while the second is about mechanical skills like playing the guitar or holding a paintbrush. Spaced repetition covers the need for declarative memory, but not for procedural memory. For the latter, intervals should not exceed a few weeks. That is why having custom deck settings would be awesome.
